src/NHibernate/AdoNet/AbstractBatcher.cs

```diff
@@ -232,7 +228,7 @@ public int ExecuteNonQuery(DbCommand cmd)
 			}
 			finally
 			{
-				if (Log.IsDebugEnabled() && duration != null)
```

Comment:
`duration` is created only under Debug logging. I see no point in additional code cluttering with  `Log.IsDebugEnabled()`